packages/pulumi-aws/src/apps/api/ApiPrerendering.ts

```diff
@@ -16,6 +16,7 @@ interface PreRenderingServiceParams {
     cognitoUserPoolArn: pulumi.Input<string>;
     awsAccountId: pulumi.Input<string>;
     awsRegion: pulumi.Input<string>;
+    elasticsearchDynamodbTableArn: pulumi.Input<string | undefined>;
```

Comment:
Can we double check... is this really needed with prerendering ?

Because, from what I can remember, prerendering service was only relying on DynamoDb, no matter which setup was used (ddb, or ddb+es).

Reply:
Maybe, I haven't verified that. That is simply rewritten from `ddb-es` template: https://github.com/webiny/webiny-js/blob/next/packages/cwp-template-aws/template/ddb-es/api/pulumi/prod/policies.ts#L90